### PR TITLE
Map component

### PR DESCRIPTION
**Pull request by jenow**
## What does this PR do ?

Add the Map shared component.

### CODE REVIEW

File: frontend/src/router.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ import Login from '@/views/Login.vue';
 import DefaultContainer from '@/views/DefaultContainer.vue';
 import PageNotFound from '@/views/404.vue';
 
+import Map from '@/views/Map.vue';
+
```
Comment (jenow) [inline frontend/src/router.ts:9]:
```suggestion
```

File: frontend/.sharedComponents/map/components/Map.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <l-map ref="map">
+      <l-tile-layer :url="url" :attribution="attribution" />
+      <slot name="marker"></slot>
+      <slot name="content"></slot>
+    </l-map>
+  </div>
+</template>
+
+<script>
+import { ref, onMounted } from '@vue/composition-api';
```
Comment (jenow) [inline frontend/.sharedComponents/map/components/Map.vue:12]:
```suggestion
import { ref } from '@vue/composition-api';
```

File: frontend/src/components/HelloWorld.vue [hunk 3]
```diff
@@ -105,4 +105,9 @@ li {
 a {
   color: #42b983;
 }
+
```
Comment (xbill82) [inline frontend/src/components/HelloWorld.vue:108]: Is it OK to have this component?

**Review by xbill82 — APPROVED** [review]

File: frontend/src/components/HelloWorld.vue (hunk 3 above)
Comment (jenow) [reply frontend/src/components/HelloWorld.vue:108]: I will probably remove it, I also wanted to check if all dependencies are really used and clean the project in another PR.

**Review by Njuelle — APPROVED** [review]

**Pull request merged**